### ISSUE TITLE
"Content Unavailable" error after unlocking vault

### ISSUE THREAD

**Issue by HelixCA** [cryptomator/ios#310]
### Please agree to the following

- [X] I have searched [existing issues](https://github.com/cryptomator/cryptomator/issues?q=) for duplicates
- [X] I agree to follow this project's [Code of Conduct](https://github.com/cryptomator/cryptomator/blob/develop/.github/CODE_OF_CONDUCT.md)

### Summary

Vault crerated under 1.70 for Mac will not open in IOS

### What software is involved?

- Operating System: Mac Montery
- Cryptomator: 1.70 DMG-4333
- …

### Volume Type

macFUSE

### Steps to Reproduce

1. [First Step] Create new Vault on OneDrive
2. [Second Step] Try to open this vault with the latest iOS app under iOS 15.7.1
3. 

### Expected Behavior

The OneDrive vault would open in Files App

### Actual Behavior

The app reports 'Content Unavailable', contents could not be displayed due to unknown error.
Tried repeatedly, same result.
None of this happened with a Vault created on OneDrive with the previous Cryptomator build.
[Logs.zip](https://github.com/cryptomator/cryptomator/files/10878379/Logs.zip)

### Reproducibility

Always

**Comment by tobihagemann** [comment]
Can you please enable Debug Mode, trigger the error, and then send us the log files to support@cryptomator.org? https://docs.cryptomator.org/en/latest/ios/settings/

If you want, review the log files yourself first and redact any sensitive information.

**Comment by HelixCA** [comment]
Hello Tobias

Thanks for your fast reply.  I am afraid that I actually addressed the problem last night and am therefore unable to send a log file.

What I did was : 

- Delete an existing OneDrive CM folder on my Mac 

- Removed the existing folder within CM on the Mac

- Delete the connection to the folder on my iPhone

- Created a new folder on OneDrive using the 1.70 build on my Mac

- Went to CM on my iPhone and tried to open the new folder.

The errors consistently started from that point and I was completely unable to view the new folder in the Files App, hence my bug report.

I went to CM on an iPad and had no problem opening the folder there.  So, I deleted CM on my iPhone and re-installed the app and the OneDrive connection.  The folder now opens every time without any issue.

Unfortunately I cannot be any more specific.  I originally thought that the problem was older iOS app with new 1.70 folder.  As it worked fine on the iPad, that was obviously not the issue.

I hope this is of some help.

Cheers

Steven

**Comment by tobihagemann** [comment]
Thank you for reporting this! I think I'm able to reproduce this issue but it's really hard to track down. It seems like a caching issue. And it's quite possible that it also affects other cloud storage services that use the same cache technique (Google Drive, pCloud). I believe the crucial step is that you removed and re-created the vault on your Mac at the same path.

So indeed, the current workaround is to somehow invalidate this internal cache. Obviously, this happens with a re-installation. But I believe it's enough to kill the Files app and to try again (maybe not, I'm not 100% sure).

**Comment by HelixCA** [comment]
Hi Tobias

The cache theory makes a lot of sense.  I had opened the ver 1.6x vault from OneDrive just before I shut it down, created the new ver 1.7x vault, and then tried to open it again, resulting in the error messages.  The devices that easily opened the 1.7x vault without error had been recently rebooted (clear cache) and I did not open the older version first.  So, I think you are onto something !

Good hunting

Steven
